app/models/equivalent/overseas_completion.rb

```diff
@@ -1,7 +1,7 @@
 module Equivalent
   class OverseasCompletion < OverseasCompletion
     def next_step
-      return "accept_privacy_policy" if confirmed
+      return "equivalent/accept_privacy_policy" if confirmed
```

Comment:
Why do we need a different privacy policy step for each branch?

Reply:
We did until I realised we can differentiate on the degree_options instead...

Reply:
we could set another degree_option value in the returning_teacher model, so the value would match the path

Reply:
I think this is disjointed; the type of candidate they are can't naturally be inferred from the privacy policy -- they're the same for each candidate. It should be linked to the logic where that decision is made, so if the candidate is on the 'equivalent' track then the candidate type of 'equivalent' is established when they select that their degree type.

Using the degree type is going to be brittle; we're creating artificial business logic and it's going to be odd/change if the privacy policy is no longer the last step in the journey.

Reply:
I have altered the candidate submission to rely on the degree_options. It needs QA and will have to rm the extra accept_privacy_policy models, extend testing for returning_teacher...

Reply:
Done